Pokemon/Pokemon/PokemonTableViewController.m

```diff
@@ -0,0 +1,506 @@
+                          @"Nidoran",
+                          @"Nidorino",
+                          @"Nidoking",
+                          @"Zubat",
+                          @"Golbat",
+                          @"Oddish",
+                          @"Gloom",
+                          @"Vileplume",
+                          @"Venonat",
+                          @"Venomoth",
+                          @"Bellsprout",
+                          @"Weepinbell",
+                          @"Victreebel",
+                          @"Tentacool",
+                          @"Tentacruel",
+                          @"Grimer",
+                          @"Muk",
+                          @"Gastly",
+                          @"Haunter",
+                          @"Gengar",
+                          @"Koffing",
+                          @"Weezing",
+                          @"Exeggcute",
+                          @"Exeggutor",
+                          @"Tangela",
+                          @"Paras",
+                          @"Parasect",
+                          @"Dratini",
+                          @"Dragonair",
+                          @"Dragonite",
+                          @"Pidgey",
+                          @"Pidgeotto",
+                          @"Pidgeot",
+                          @"Rattata",
+                          @"Raticate",
+                          @"Spearow",
+                          @"Fearow",
+                          @"Jigglypuff",
+                          @"Wigglytuff",
+                          @"Meowth",
+                          @"Persian",
+                          @"Farfetch",
+                          @"Doduo",
+                          @"Dodrio",
+                          @"Lickitung",
+                          @"Chansey",
+                          @"Kangaskhan",
+                          @"Tauros",
+                          @"Ditto",
+                          @"Eevee",
+                          @"Porygon",
+                          @"Snorlax",
+                          @"Geodude",
+                          @"Graveler",
+                          @"Golem",
+                          @"Onix",
+                          @"Rhyhorn",
+                          @"Rhydon",
+                          @"Aerodactyl",
+                          @"Clefairy",
+                          @"Clefable",
+                          @"Mankey",
+                          @"Primeape",
+                          @"Machop",
+                          @"Machoke",
+                          @"Machamp",
+                          @"Hitmonlee",
+                          @"Hitmonchan"];
+
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+#warning Potentially incomplete method implementation.
```

Comment:
You can remove this line (and thus solve the warning it creates) once you've implemented the method.